db/migrate/20191017185340_add_unique_constraint_to_profile_by_account.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueConstraintToProfileByAccount < ActiveRecord::Migration[5.2]
+  def change
+    add_index(:profiles , %i[account_id name benchmark_id], unique: true)
```

Comment:
Also, this doesn't match the model. Name doesn't matter, and ref_id does: https://github.com/RedHatInsights/compliance-backend/blob/master/app/models/profile.rb#L17

```suggestion
    add_index(:profiles , %i[account_id ref_id benchmark_id], unique: true)
```

Reply:
Good catch, the commit was right before https://github.com/RedHatInsights/compliance-backend/commit/8bf4510eca6bcc955ef28ae4ede2fb12607980f6#diff-6ef5e2d67a2bf4ad80d150039dccd3f2L17 - I didn't realize the name constraint was removed

Reply:
Unique name was essentially a stop-gap solution to the differences between benchmark versions. Now we don't need that anymore! :champagne: